types/wnumb/index.d.ts

```diff
@@ -1,8 +1,12 @@
-// Type definitions for wnumb 1.0
+// Type definitions for wnumb 1.2
 // Project: https://github.com/leongersen/wnumb
 // Definitions by: Corey Jepperson <https://github.com/acoreyj>
+//                 Jamie Neubert Pedersen <https://github.com/eikooc>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+export default wNumb;
+export as namespace wNumb;
```

Comment:
I think you want to remove the `declare namespace` below so you have

`export as namespace wNumb`
`export interface Options`
`export interface Instance`

Reply:
Alright. Latest commit should do that. Thanks!

Reply:
You do need to export those interface if you want that type info for intelliSense, etc:

```ts
export interface Options {
    decimals?: number;
    ...
}

export interface Instance {
    ...
}
```

You should modify the test to use these types to make sure it actually works:

```ts
const options: wNumb.Options = {decimals: 1}
const moneyFormat: wNumb.Instance = wNumb(options);
```

Otherwise your types will just show up as `any` which is probably not what you want.

Reply:
Okay, I didn't catch that. Tests and types should be updated now :)